scripts/spectrum-vars.js

```diff
@@ -113,38 +135,55 @@ const scales = ['medium', 'large'];
 const cores = ['global'];
 const processes = [];
 
-themes.forEach(async (theme) => {
-    const srcPath = path.join(spectrumPath, `spectrum-${theme}.css`);
-    const dstPath = path.resolve(
-        path.join(__dirname, '..', 'packages', 'styles', `theme-${theme}.css`)
-    );
+spectrumPaths.forEach(async (spectrumPath, i) => {
+    const packageDir = ['styles'];
+    if (i == 1) packageDir.push('express');
+    themes.forEach(async (theme) => {
+        if (i == 1 && (theme === 'lightest' || theme === 'darkest')) return;
```

Comment:
I'd worry about selecting on the index here except that the number of spectrum paths changes so slowly, but maybe we could make it self-documenting:

```suggestion
    const isExpress = i === 1;
    if (isExpress) packageDir.push('express');
    themes.forEach(async (theme) => {
        if (isExpress && ['lightest', 'darkest'].includes(theme)) return;
```